benchmarks/distributed/rpc/parameter_server/experiment_scripts/ddp_cpu_sparse_rpc_nccl_allreduce.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cd "$(dirname "$0")"
+cd ..
+
+python -u launcher.py \
+    --benchmark="4" \
```

Comment:
curious, any reason for using numbers to for `--benchmark` instead of allowing users to directly specify `timeout`, `batch_size`, etc.?

Reply:
no reason, the benchmark config can be removed since it all runs use the same arguments.